### DISCUSSION TITLE
RainbowKit cannot be used in a React project?

### DISCUSSION THREAD

**Discussion by linghuccc** [rainbow-me/rainbowkit#1325] [Q&A] [answered]
I am not sure if this should be an issue, but I will just put is in discussion first.

I tried to initialize a React project by running command: npx create-react-app my-app. Then I followed instructions in rainbowKit website to install rainbowKit: npm install @rainbow-me/rainbowkit wagmi viem. I got below error: 
![rainbow](https://github.com/rainbow-me/rainbowkit/assets/59200993/68f8a5e3-8255-4d39-967a-6d7f3809f92d)

Is this an issue of rainbowKit?

**Comment by glitch-txs** [comment]
create-react-app was deprecated, maybe consider using a framework or vite 🤔

**Reply by linghuccc** [reply]
OK, acknowledged. Thank you.

**Comment by DanielSinclair** [comment, marked as answer]
We do offer a Create React App example [here](https://github.com/rainbow-me/rainbowkit/tree/main/examples/with-create-react-app). In recent months, because of changes to Webpack and Wagmi/Viem/WalletConnect, you will need to do more massaging to the configuration as documented [here](https://www.rainbowkit.com/docs/installation#react-nextjs-and-webpack)
